tf2onnx/graph.py

```diff
@@ -252,6 +257,22 @@ def update_proto(self):
         if attr:
             self._op.attribute.extend(attr)
 
+    def build_onnx_node(self):
```

Comment:
seems it is only used in infer_onnx_shape_dtype?
move to utils or a inline function of infer_onnx_shape_dtype?